### DISCUSSION TITLE
Buffering while playing YouTube video

### DISCUSSION THREAD

**Discussion by kamyogi** [smplayer-dev/smplayer#247] [Q&A] [answered]
HI,

Its buffering while watching videos from YouTube.

While checking for Updates, it shows 206.0.9418 as installed one and available version is 21.8.0.9887. How to update the version since there's no prompt for this? Only asking to know more on the new version.

While updating YouTube code (through 'Install/Update YouTube support' , it updates youtube-dl to 2021.06.06 but the buffering continues!

What could be wrong?

Thanks for participating.

**Comment by smplayer-dev** [comment, marked as answer]
Go to https://github.com/smplayer-dev/smplayer/releases/tag/v21.10.0 and dowload the latest version. Then go to _preferences -> network_ and select **yt-dlp** instead of **youtube-dl** as youtube application. See if that fixes the youtube problem for you.

Edit: updated the release link.

**Reply by kamyogi** [reply]
Thanks smplayer-dev for your reply. One more thing that I forgot to mention in my previous mail that I am using SMPlayer on Ubuntu 20.04.3 LTS. Which file to take and how to install it?
Thanks for your reply.

**Reply by smplayer-dev** [reply]
This package is for Ubuntu 20: [smplayer_21.8.0.9999-1.ubuntu-focal_amd64.deb](https://github.com/smplayer-dev/smplayer/releases/download/v21.8.0.9999/smplayer_21.8.0.9999-1.ubuntu-focal_amd64.deb)

To install it:
`sudo dpkg -i smplayer_21.8.0.9999-1.ubuntu-focal_amd64.deb`

**Reply by kamyogi** [reply]
It works! :=) Thankyou very much smplayer-dev.

**Comment by hpmc13** [comment]
Thank you! This has been driving me nuts for a few days now. Finally stumbled on this and it works. Back to beautiful videos without YouTube.
